circuits/cpp/src/aztec3/circuits/abis/contract_deployment_data.hpp

```diff
@@ -102,6 +104,7 @@ template <typename NCT> void read(uint8_t const*& it, ContractDeploymentData<NCT
 {
     using serialize::read;
 
+    read(it, data.deployer_public_key);
     read(it, data.constructor_vk_hash);
```

Comment:
... TBH I wonder if a practical measure for now is to also derive this from msgpack. It would give some wins before moving fully over